src/components/IPFS/Config.js

```diff
@@ -0,0 +1,25 @@
+import temporallogo from '../../assets/temporal.png'
+
+const TEMPORAL = {
+  name: 'Temporal', // Name of the IPFS service
+  logo: temporallogo, // Logo used for the login/sign up header
+  host: 'api.ipfs.temporal.cloud', // the IPFS api endpoint used when the etherium profile is set to mainnet
+  dev: 'dev.api.ipfs.temporal.cloud', // the IPFS api endpoint used when not on mainnet
+  port: '443', // the port the endpoint is served from
+  apiPath: '/api/v0/', // Api path extension if nessecary
+  protocol: 'https', // the protocol used to comunicate with the endpoint
+  auth: true, // If the api requires authention headers
+  login: 'https://api.temporal.cloud/v2/auth/login', // the Authentication endpoint to verify users information. Currently supports JWT
+  loginDev: 'https://dev.api.temporal.cloud/v2/auth/login', // same as login but used if user is not on mainnet
+  signup: 'https://api.temporal.cloud/v2/auth/register',
+  signupDev: 'https://dev.api.temporal.cloud/v2/auth/register'
+}
+
+export function getConfig(service) {
+  switch (service) {
+    case (service = 'TEMPORAL'):
+      return TEMPORAL
+    default:
+      return TEMPORAL
```

Comment:
Why are they returning `TEMPORAL` in default as well as service = 'TEMPORAL'?

Reply:
Technically we don't really need this right now but later when adding other services this could be a useful helper function. Got rid of the default in be67c21 but maybe get rid of `service === TEMPORAL` or the whole thing?